app/views/coronavirus_local_restrictions/_devolved_nation.html.erb

```diff
@@ -2,16 +2,19 @@
   title: t("coronavirus_local_restrictions.results.devolved_nations.heading")
 } %>
 <% if @location_lookup.data.first.scotland? %>
-  <p class="govuk-body">
+  <p class="govuk-body govuk-!-margin-bottom-9">
```

Comment:
It adds visual spacing between the two sections, I've added a screenshot below to give more context:

![image (1)](https://user-images.githubusercontent.com/71266765/96136049-9ab00200-0ef3-11eb-954d-eeed74c7459c.png)

Yes, these files could be refactored to be more DRY yet keep a balance between reactive updates that need to go live ASAP (and change often).  

If this draft PR doesn't conflict drastically to parallel work that being done at your end I can build upon.

Reply:
Can a margin-top be added instead?

Reply:
Not directly on the component but yes this can be improved, is there any draft work that's been pushed to review how this mixes in in terms of scaffolding?

Reply:
Can it be added to the "_changes_information" partial?

Reply:
This was updated to be contained within the partial.